composer.json

```diff
@@ -12,24 +12,25 @@
         }
     ],
     "require": {
-        "symfony/framework-bundle": "~2.1",
-        "symfony/process": "~2.1",
-        "symfony/finder": "~2.1",
+        "php": "^5.5|~7.0",
+        "symfony/framework-bundle": "~2.8|~3.0",
```

Comment:
2.3 it's not BC from 3.0, but we can lower the requirement to 2.6. Below that, we don't have TokenStorage and Authorization checker, that are the replacement of the SecurityStorage, that is deprecated since 2.6 and absent from 3.0.
Support for 2.3 will be there with older versions of this bundle.